.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+*       @lacework/eng-product-platform
```

Comment:
Maybe `@lacework/awls-devs` too?

Reply:
Agreed! @leijin-lw pointed this out on the other agentless repos as well. Will update